low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml

```diff
@@ -0,0 +1,36 @@
+cluster:
+   product: LDR_R2
+   cluster_id: CC01
+   server_nodes:
+     abc25957545942cfa728b74cead1a882: srvnode1
+   srvnode1:
+      node_type: physical
+      site_id: DC01
+      rack_id: RC01
+      node_id: SN01
+      storage_set_id: ST01
+      operating_system: centos7
```

Comment:
this would go away, need to read from /etc/system-release

Reply:
Done.